### PR TITLE
Sponsors section

### LINKED ISSUE #149: Sponsors Section - DESKTOP
- [ ] Implement sponsors section on desktop
Screenshot 2023-02-01 at 12 06 39 PM

### PR DESCRIPTION
**Pull request by jzhzhang66**
Sponsors section desktop

Fixes #149 .

Changelist (WEEK 1):

- Created a SponsorsSection component to contain all of the code for the sponsors section (only desktop is implemented in this ticket) 
- NOTE: I'm waiting to hear back from sponsib about the packet link so currently the button doesn't link anywhere but I'll add that in as soon as I hear back.
- There was a duplicate typography file for some reason so I deleted it.

Changelist (WEEK 2): 
- Sponsors section on mobile 
- Fixed cases where the cars in the race section weren't on the line 
- Added more spacing between sections 
- Added "happening now" to appear whenever an event is happening right now in the coming up section 
- Added the timer to the bottom of the live site 

Tested:

- Chrome and Safari

Screenshots & Screencasts:
Screenshot 2023-02-07 at 9 07 51 PM
Screenshot 2023-02-07 at 9 07 28 PM
https://user-images.githubusercontent.com/59738880/217410770-41ece409-7532-40af-8d6c-911ab2d0e9bc.mov

### CODE REVIEW

File: main-site/components/sponsors-section/SponsorsSection.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        <br />
+        <StyledGoldLogos src={GoldLogosDesktop} alt="gold level logos" />
+      </div>
+      <div>
+        <SecondaryButton btnText="Silver Level" isClickable={false} />
+        <br />
+        <StyledSilverLogos src={SilverLogosDesktop} alt="silver level logos" />
+      </div>
+      <StyledContactText>
+        Interested in joining the HackBeanpot universe? <br /> <br /> Reach out
```
Comment (surbhi-gulati) [inline main-site/components/sponsors-section/SponsorsSection.tsx]: this seems like the space theme wording from last year? do we want to keep it

**Review by surbhi-gulati — APPROVED** [review]
Approved with question to confirm wording

**Review by Dhivas-Sugumar — APPROVED** [review]
LGTM!!

File: main-site/components/sponsors-section/SponsorsSection.tsx (hunk 1 above)
Comment (jzhzhang66) [reply main-site/components/sponsors-section/SponsorsSection.tsx]: nice catch! I'll change it to desert exploration

**Review by Dhivas-Sugumar — APPROVED** [review]
LGTM :SALUTE:

**Pull request merged**